ui_testing/testcases/basic_tests/test08_contacts.py

```diff
@@ -0,0 +1,382 @@
+        first_contact_data = self.base_selenium.get_row_cells_dict_related_to_header(row=created_contact_record)
+
+        self.base_selenium.LOGGER.info('contact no is {}, and it should be {}'.format(first_contact_data['Contact No'], contact_data['no']) )
+        self.assertEqual(first_contact_data['Contact No'], contact_data['no'])
+
+        self.base_selenium.LOGGER.info('contact name is {}, and it should be {}'.format(first_contact_data['Contact Name'], contact_data['name']) )
+        self.assertEqual(first_contact_data['Contact Name'], contact_data['name'])
+
+        self.base_selenium.LOGGER.info('contact address is {}, and it should be {}'.format(first_contact_data['Address'], contact_data['address']) )
+        self.assertEqual(first_contact_data['Address'], contact_data['address'])
+
+        self.base_selenium.LOGGER.info('contact postalcode is {}, and it should be {}'.format(first_contact_data['Postal Code'], contact_data['postalcode']) )
+        self.assertEqual(first_contact_data['Postal Code'], contact_data['postalcode'])
+
+        self.base_selenium.LOGGER.info('contact location is {}, and it should be {}'.format(first_contact_data['Location'], contact_data['location']) )
+        self.assertEqual(first_contact_data['Location'], contact_data['location'])
+        
+        self.base_selenium.LOGGER.info('contact country is {}, and it should be {}'.format(first_contact_data['Country'], contact_data['country']) )
+        self.assertEqual(first_contact_data['Country'], contact_data['country'])
+
+        self.base_selenium.LOGGER.info('contact email is {}, and it should be {}'.format(first_contact_data['Email'], contact_data['email']) )
+        self.assertEqual(first_contact_data['Email'], contact_data['email'])
+
+        self.base_selenium.LOGGER.info('contact phone is {}, and it should be {}'.format(first_contact_data['Phone'], contact_data['phone']) )
+        self.assertEqual(first_contact_data['Phone'], contact_data['phone'])
+
+        self.base_selenium.LOGGER.info('contact skype is {}, and it should be {}'.format(first_contact_data['Skype'], contact_data['skype']) )
+        self.assertEqual(first_contact_data['Skype'], contact_data['skype'])
+
+        self.base_selenium.LOGGER.info('contact website is {}, and it should be {}'.format(first_contact_data['Website'], contact_data['website']) )
+        self.assertEqual(first_contact_data['Website'], contact_data['website'])        
+
+        self.base_selenium.LOGGER.info('contact departments is {}, and it should be {}'.format(first_contact_data['Departments'], contact_data['departments']) )
+        self.assertEqual(first_contact_data['Departments'], contact_data['departments'])
+
+        self.base_selenium.LOGGER.info('contact contact_type is {}, and it should be {}'.format(first_contact_data['Type'], contact_data['contact_type']) )
+        self.assertEqual(first_contact_data['Type'], contact_data['contact_type'])
```

Comment:
very much true, i am using this function (get_full_contact_data), in many other cases, so it is easier to right short simple names instead of full header name, and i need to compare the data with the table's in this case only.